src/css/styles.scss

```diff
@@ -1,140 +1,108 @@
+// COLORS VARS
+:root {
+  --bcg-color: #111f28;
+  --section-bcg-color: #484e52;
+  --base-color: #ffffff;
+  --accent-color: #dda0dd;
+}
+
+$base-font: 'Open Sans', sans-serif;
+
 * {
+  border: none;
   box-sizing: border-box;
-  font-family: 'Open Sans', sans-serif;
+  font-family: $base-font;
   margin: 0;
   padding: 0;
 }
+
 body {
-  background-color: #111f28;
+  background-color: var(--bcg-color);
```

Comment:
Because you are using Sass, you should just use the Sass variable syntax (as you did for the `$base-font` variable) rather than using traditional CSS vars. This is just a consistency thing -- the Sass variable syntax is a little nicer/cleaner and if you have access to it, you might as well use it.

Reply:
We use css properties for easy switch between dark mode and light mode, because we can't manipulate with sass variables using JS. CSS properties allows us to do it.